plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/view/dev/Stores.js

```diff
@@ -33,56 +35,64 @@ Ext.define('NX.view.dev.Stores', {
     var me = this;
 
     Ext.apply(me, {
```

Comment:
/me shrugs, its a mixed bag IMO, depends on if you prefer foo: bar or me.foo=bar style mostly.